app/models/project/Project.scala

```diff
@@ -34,21 +29,54 @@ case class ProjectSQL(
                      expectedTime: Option[Long],
                      created: Long = System.currentTimeMillis(),
                      isDeleted: Boolean = false
-                     )
+                     ) extends FoxImplicits {
```

Comment:
Here we could just use the implicit `_id: BSONObjectID = BSONObjectID.generate` like in `Project` before.
And could `_team` and `_owner` just be the actual Objects instead of `id`s?

Reply:
I found the implicit id generation a bit problematic, since it happened sometimes that new objects were created with random IDs where actually old IDs should have been used. So let’s make this explicit in the very few cases where actual new objects are created.

_team and _owner being the IDs relates to what I wrote above. We do not want to query the team and user sql tables everytime we fetch Project information.

Reply:
also, the implicit id generation requires _id to be among the last fields (and that would not match our order in the schema anymore, which I don’t like :D)